MIGRATION_GUIDE.md

```diff
@@ -12,6 +12,16 @@ In recent changes, we introduced a new grant resources to replace the old ones.
 To aid with the migration, we wrote a guide to show one of the possible ways to migrate deprecated resources to their new counter-parts.
 As the guide is more general and applies to every version (and provider), we moved it [here](./docs/technical-documentation/resource_migration.md).
 
+## v0.85.0 ➞ v0.86.0
```

Comment:
you will have to resolve conflicts here, because there was already started `v0.85.0 ➞ v0.86.0` in the meantime changes